backend/src/main/java/de/neuefische/backend/service/AccuWeatherApiService.java

```diff
@@ -0,0 +1,23 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.DailyForecasts;
+import de.neuefische.backend.model.WeatherForecast;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Objects;
+
+@Service
+public class AccuWeatherApiService {
+
+    private static final String API_URL = "http://dataservice.accuweather.com/forecasts/v1/daily/1day/178087?apikey=xiXJGJfVofWBtkbSj51vTclQh0ognRyA&details=true&metric=true";
```

Comment:
don't publish sensitive data, such as API keys, to github. Create an env variable instead

Reply:
Already done in the following branch